snuba/subscriptions/scheduler_processing_strategy.py

```diff
@@ -251,6 +255,17 @@ def submit(self, message: Message[Tick]) -> None:
             self.__next_step.submit(self.__buffers[tick_partition].popleft())
             return
 
+        # Periodically record the legnth of each buffer
+        now = time.time()
+        if now - self.__last_recorded_time > self.__record_frequency_seconds:
+            for partition_index in self.__buffers:
+                self.__metrics.gauge(
```

Comment:
Is the metric already partitioned by dataset or entity over here? If not, then please add it

Reply:
Already done at the top before the metrics object is passed in here

Reply:
Can you show me the line? I don't see that anywhere

Reply:
https://github.com/getsentry/snuba/blob/e1f0764c5b1f89552a2eda57d2d36bca7c8ae0e3/snuba/cli/subscriptions_scheduler.py#L111-L113